tests/console/runc.pm

```diff
@@ -23,81 +23,9 @@ use strict;
 
 sub run {
     select_console("root-console");
+    
+    # TODO
 
-    my $runc = is_sle && sle_version_at_least('15') ? 'docker-runc' : 'runc';
```

Comment:
Why not just drop this test entirely for now?

Reply:
sure, it makes sense. Let me do this.